recipes/scala/src/main/scala/com/adobe/platform/ml/helper/Helper.scala

```diff
@@ -19,9 +19,7 @@ package com.adobe.platform.ml
 
 import java.time.LocalDateTime
 
-import com.adobe.platform.dataset.DataSetOptions
```

Comment:
Why did we remove this.  We should be using their constants wherever possible.

Reply:
Sure, I will add the QSOption instead of DataSetOptions. That is the new object in query service sdk

Reply:
no you need both

Reply:
I do see that change https://github.com/adobe/experience-platform-dsw-reference/pull/105/files#diff-32910ed8f324dcd3946665f8dbc869bfR96 thanks!